style/app.scss

```diff
@@ -2524,3 +2524,8 @@ main-filter input
   position         : absolute;
   padding          : 10px;
 }
+
+.show-dropdown-arrow::after
```

Comment:
Okay, so let's make this look a bit nicer by

1. adding some padding between the arrow and the link text
2. making the arrow a smaller font
3. making the arrow also change colour when the link is hovered over